### PR TITLE
Add fulfilled Mask count on Summary page

### LINKED ISSUE #22: Add Fourth Status Value
The fourth value can be "fulfilled" number.  It can just be a sum of the all requested numbers (both sizes) when the tag "fulfilled" is set to true.

### PR DESCRIPTION
**Pull request by devarsh19**
Add count for fulfilled Mask Requests when `requestFulfilled = true `
Screen Shot 2022-01-29 at 8 16 50 PM

 I checked db - currently it seems all records have either `requestFulfilled = false ` or don't have flag field added. So count is showing 0. For testing purpose I have toggled couple of entries to verify the fulfilled Mask count

closes #22

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -16,10 +16,12 @@ app.use(bodyParser.urlencoded({ extended: true }));
 // get driver connection
 const dbo = require("./db/conn");
 
-https.createServer({
-  key: fs.readFileSync('privatekey.pem'),
-  cert: fs.readFileSync('fullchain.pem'),
-}, app).listen(port, () => {
+// https.createServer({
```
Comment (devarsh19) [inline server/server.js:19]: @HaiderZaidiDev I have commented these lines - I am not sure do we need this?  as its giving me error for these two files not found
 - `privatekey.pem`
 - `fullchain.pem`

**Comment by mekkim** [comment]
Looks good!

**Review by mekkim — APPROVED** [review]
Looks fine to me visually. I note some cleaning up of the code (awesome!) which should be fine if it tests fine.  Great work!

File: server/server.js (hunk 1 above)
Comment (mekkim) [reply server/server.js:19]: I'm not sure what these do.  Are they related to the MongoDB API perhaps?

Comment (HaiderZaidiDev) [reply server/server.js:19]: Hi @devarsh19, we do need these, as they're required for the API server to run in SSL when in production. It's okay to have it out in development but it's needed for the prod branch.

Comment (devarsh19) [reply server/server.js]: oh okay, Thanks for the information @HaiderZaidiDev

**Pull request merged**